pkg/cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -41,22 +42,33 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		Short: "Deletes a cluster",
 		Long:  "Deletes a resource",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return runE(logger, flags)
+			return deleteClusters(logger, flags)
 		},
 	}
 	cmd.Flags().StringVar(&flags.Name, "name", cluster.DefaultName, "the cluster name")
 	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", "", "sets kubeconfig path instead of $KUBECONFIG or $HOME/.kube/config")
+	cmd.Flags().BoolVar(&flags.All, "all", false, "delete all clusters")
 	return cmd
 }
 
-func runE(logger log.Logger, flags *flagpole) error {
-	// Delete the cluster
-	logger.V(0).Infof("Deleting cluster %q ...\n", flags.Name)
+func deleteClusters(logger log.Logger, flags *flagpole) error {
 	provider := cluster.NewProvider(
 		cluster.ProviderWithLogger(logger),
 	)
-	if err := provider.Delete(flags.Name, flags.Kubeconfig); err != nil {
-		return errors.Wrap(err, "failed to delete cluster")
+
+	if !flags.All {
+		// Delete individual cluster
+		logger.V(0).Infof("Deleting cluster %q ...\n", flags.Name)
```

Comment:
the original had the `\n`, but i think it can be deleted.
logger entries are per new line. [1]

Reply:
Fixed in latest push